### PR TITLE
Preware v1.9.16

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,7 +7,6 @@ src/ipkgservice
 tags
 nppBackup
 .DS_Store
-bin/*
```
Comment (Herrie82) [inline .gitignore]: This was there for a reason. The ipkgservice is a binary that is depending on CPU architecture (x86, x64, armv7, aarch64 etc), so it cannot be included like this!

Comment (ghost) [reply .gitignore]: mhhhhh i didnt saw it

**Comment by rwhitby** [comment]
Is the intention here to create a new pull request?

**Comment by ghost** [comment]
Yeah for the new version but i closed because i deleted My fork but dont worry i Will refork the repo put the file and merge it

**Pull request closed without merging**